tests/test_wfgenerator.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import os
+from .util import get_data, windows_needs_docker
+from cwltool.main import main
+
+@windows_needs_docker
+def test_missing_enable_ext():
+    # Requires --enable-ext and --enable-dev
+    if "CWLTOOL_OPTIONS" in os.environ:
+        del os.environ["CWLTOOL_OPTIONS"]
```

Comment:
Probably should restore this at the end of the test, to avoid side effects

Reply:
Yes, you're right.